### ISSUE TITLE
update call to dropout with deprecated keep_prob

### ISSUE THREAD

**Issue by cwmeijer** [NLeSC/mcfly#191]
WARNING:tensorflow:From ../keras/backend/tensorflow_backend.py:3445: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.

**Comment by aishgrt1** [comment]
Any updates?
